.github/workflows/tests.yml

```diff
@@ -22,7 +22,7 @@ jobs:
           go-version: ${{ matrix.go-version }}
 
       - name: Go build (source)
-        run: make
+        run: make build
 
       - name: Go build (tests)
```

Comment:
Do we want to run tests in pipeline for both versions? 1.16 and 1.20? The original task for unit tests required tests to be run in container, right now the make command has image with 1.20 golang hard coded, it is necessary to change that if we want to run tests for 1.16

Reply:
Ah, good observation! I would say yes, that would be nice to have. 

I will make the changes to the container.

Reply:
done!